src/ripple/app/main/GRPCServer.h

```diff
@@ -151,7 +151,7 @@ class GRPCServerImpl final
         grpc::ServerContext ctx_;
 
         // true if finished processing request
-        std::atomic_bool finished_;
```

Comment:
`isFinished` is part of the public interface and accesses this variable. Can we really say this will always be accessed from the same thread? If we can, we should at least add a comment as-to why.

Reply:
@seelabs I see your concern. The variable should never need to be accessed from a different thread. I am not really sure how to ensure this is always the case, but I don't like making this atomic when it doesn't need to be. Can we just add a comment above isFinished() saying it is not threadsafe?

Reply:
(Just adding a comment; CJ and I spoke via DM): I think we decided to add the `atomic` back.